<!DOCTYPE HTML>
<!--
	Camille Kessler
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Camille Kessler</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
				<header id="header">
					<h1><a href="index.html">Camille Kessler</a></h1>
					<nav class="links">
						<ul>
							<li><a href="me.html">About me</a></li>
							<li><a href="phd.html">PhD project</a></li>
							<li><a href="master.html">Master's thesis</a></li>
							<li><a href="other.html">Other</a></li>
						</ul>
					</nav>
					<nav class="main">
						<ul>
							<li class="search">
								<a class="fa-search" href="#search">Search</a>
								<form id="search" method="get" action="#">
									<input type="text" name="query" placeholder="Search" />
								</form>
							</li>
							<li class="menu">
								<a class="fa-bars" href="#menu">Menu</a>
							</li>
						</ul>
					</nav>
				</header>

				<!-- Menu -->
				<section id="menu">

					<!-- Search -->
						<section>
							<form class="search" method="get" action="#">
								<input type="text" name="query" placeholder="Search" />
							</form>
						</section>

					<!-- Links -->
						<section>
							<ul class="links">
								<li>
									<a href="me.html">
										<h3>About me</h3>
									</a>
								</li>
								<li>
									<a href="phd.html">
										<h3>PhD project</h3>
										<p>North american deer speciation genomics</p>
									</a>
								</li>
								<li>
									<a href="master.html">
										<h3>Master's thesis</h3>
										<p>Alpine ibex sequencing assay and immunogenetics</p>
									</a>
								</li>
								<li>
									<a href="other.html">
										<h3>Other</h3>
									</a>
								</li>
							</ul>
						</section>
				</section>

				<!-- Main -->
					<div id="main">

						<!-- Post -->
							<article class="post">
								<header>
									<div class="title">
										<h2><a>PhD project</a></h2>
										<p>North american deer speciation genomics</p>
									</div>
									<div class="meta">
										<time class="published" datetime="2023-10-03">9 October 2023</time>
									</div>

								</header>									

								<p>White-tailed deer (<i>Odocoileus virginianus</i>; WTD) and mule deer (<i>Odocoileus hemionus</i>; MD) 
									are sister species of North American Cervidae that hybridise in areas of sympatry. They are highly 
									abundant throughout their range and are important on several aspects: First, they have been, and continue to be, 
									important to indigenous communities as a source of food, clothing and culture. Further, their high abundance
									is heavily monetised through hunting-related activities, but also impacts predator-prey dynamics and 
									vegetation. The speciation and demographic history of <i>Odocoileus</i> remained poorly understood, 
									mainly due to the unreliable identification of fossils and outdated molecular clock divergence time estimates. 
									The aim of my thesis is to examine the speciation and demographic history of <i>Odocoileus</i> using genome 
									scale data and ancient DNA.</p>
							</article>
						
						<!-- Post -->
							<article class="post">
								<header>
									<div class="title">
										<h2><a>Speciation without gene-flow in hybridising deer</a></h2>
										<p>Chapter one</p>
									</div>
									<div class="meta">
										<time class="published" datetime="2023-10-03">9 October 2023</time>
									</div>
								</header>									

								<p>The first chapter of my PhD focussed on WTD/MD hybridisation and speciation process. There is ample 
								modern evidence that both species hybridise in areas of sympatry but the lack of range-scale and genome-wide
								analysis has made it difficult to date the onset of the hybridisation. Gene-flow was thought to be
								as old as speciation and we therefore wanted to see if ecological speciation was at work in this system. 
								Surprisingly, we found that gene-flow between WTD & MD was the result of recent secondary contact, as such, 
								it did not impact the speciation process. The paper was published in <i>Molecular ecology</i>,
								here is the full abstract:
								
								<p>Under the ecological speciation model, divergent selection acts on ecological differences between 
								populations, gradually creating barriers to gene flow and ultimately leading to reproductive isolation. 
								Hybridisation is part of this continuum and can both promote and inhibit the speciation process. Here, we 
								used white‐tailed (<i>Odocoileus virginianus</i>) and mule deer (<i>O. hemionus</i>) to investigate patterns of speciation 
								in hybridising sister species. We quantified genome‐wide historical introgression and performed genome 
								scans to look for signatures of four different selection scenarios. Despite ample modern evidence of 
								hybridisation, we found negligible patterns of ancestral introgression and no signatures of divergence 
								with gene flow, rather localised patterns of allopatric and balancing selection were detected across the 
								genome. Genes under balancing selection were related to immunity, MHC and sensory perception of smell, the 
								latter of which is consistent with deer biology. The deficiency of historical gene‐flow suggests that 
								white‐tailed and mule deer were spatially separated during the glaciation cycles of the Pleistocene and 
								genome wide differentiation accrued via genetic drift. Dobzhansky‐Muller incompatibilities and selection 
								against hybrids are hypothesised to be acting, and diversity correlations to recombination rates suggests 
								these sister species are far along the speciation continuum.</p>
								 
								<footer>
									<ul class="actions">
										<li><a href="http://dx.doi.org/10.1111/mec.16824" class="button large">Read the paper</a></li>
									</ul>
								</footer>
							</article>
	
						<!-- Post -->
							<article class="post">
								<header>
									<div class="title">
										<h2><a>Genomic analyses capture the human-induced demographic collapse and recovery in a wide-ranging cervid</a></h2>
										<p>Chapter two</p>
									</div>
									<div class="meta">
										<time class="published" datetime="2023-10-03">9 October 2023</time>
									</div>
								</header>									

								<p>The second chapter of my PhD investigated the demographic history of WTD & MD and the impact of climatic
									and anthropogenic pressures on deer populations. Species demographies have been strongly shaped by climatic
									pressures such as the Last Glacial Maximum (LGM), but also more recently by anthropogenic factors. WTD & MD 
									survived in different refugia during the LGM and census data estimations suggest populations were severely 
									reduced after the European colonisation. We assumed both events had dramatic effect on deer's effective 
									population size (Ne) so we reconstructed and modelled WTD & MD's demographic histories through time.
									We found that WTD population size dramatically declined during the LGM and after European colonisation. 
									The latter is true for MD as well in addition with patterns of potential conservation concern for that 
									species. The paper is in review, it is currently only  available as preprint on
									BioRxiv, here is the full abstract:
								
								<p>The glacial cycles of the Quaternary heavily impacted species through successions of population contractions 
									and expansions. Similarly, population demography has been intensely shaped by human pressures such as 
									unregulated hunting and land use changes. White-tailed and mule deer survived in different refugia through 
									the Last Glacial Maximum, and their populations were severely reduced after the European colonisation. 
									Here, we analysed 73 re-sequenced deer genomes form across their North American range to understand the 
									consequences of climatic and anthropogenic pressures on deer demographic and adaptive history. We found a 
									strong signal of glacial-induced vicariance and demographic decline; notably, there is a severe decline in 
									white-tailed deer effective population size (Ne) at the end of the Last Glacial Maximum. We found robust 
									evidence for a colonial impact in the form of a recent and dramatic drop in Ne in all analysed populations. 
									Historical census size and restocking data show a clear parallel to historical Ne estimates, but temporal 
									Ne/Nc ratio shows patterns of conservation concern for mule deer. Signatures of selection highlight genes 
									related to temperature, including a cold receptor previously highlighted in woolly mammoth. We also detected 
									numerous immune-genes that we surmise reflect the changing land-use patterns in North America. Our study 
									provides a detailed picture of anthropogenic and climatic-induced decline in deer diversity, and clues to 
									understanding the conservation concerns of mule deer and the successful demographic recovery of white-tailed 
									deer.</p>
								 
								<footer>
									<ul class="actions">
										<li><a href="http://dx.doi.org/10.1101/2023.07.19.549627" class="button large">Read the preprint</a></li>
									</ul>
								</footer>
	
							</article>

						<!-- Post -->
							<article class="post">
								<header>
									<div class="title">
										<h2><a><i>Torontoceros hypogaus</i> ancient DNA</a></h2>
										<p>Chapter three</p>
									</div>
									<div class="meta">
										<time class="published" datetime="2023-10-03">9 October 2023</time>
									</div>
								</header>									

								<p>In progress</p>
							</article>	

						<!-- Post -->
							<article class="post">
								<header>
									<div class="title">
										<h2><a>Ancient DNA</a></h2>
										<p>Chapter four</p>
									</div>
									<div class="meta">
										<time class="published" datetime="2023-10-03">9 October 2023</time>
									</div>
								</header>									
				
								<p>In progress, here take a funny picture of me while you wait</p>
								<a class="image featured"><img src="images/me_aDNA.jpeg" 
									alt="picture of me in full body suit for ancient DNA lab work, I have the body suit, a surgical
									mask, my glasses, a helmet on top, I am seated in front of a lab hood with a magnifier and pipettes, in 
									the backround there is a partial jurassic park poster with T-rex mouth open that is probably going to eat 
									someone" 
									style="width: 40%; height: 40%"  /></a>					
						</article>
					<!-- Post -->
						<article class="post">
							<header>
								<div class="title">
									<h2><a>Island demographics and trait associations in white-tailed deer</a></h2>
									<p>Side project</p>
								</div>
								<div class="meta">
									<time class="published" datetime="2023-10-03">9 October 2023</time>
								</div>
							</header>									
							<a class="image"></a><img src="images/SPM_shore.jpeg" style="float: right; width: 40%; height: 40%; padding: 1%;"
							alt=""/></a>
							<p>This side project was initiated by the Saint-Pierre et Miquelon (SPM) authorities because they were seeing more 
								and more deer with leucism and malloclusions and were worried the cause might be inbreeding. As the same was true 
								for Florida Key deer which are much smaller than their mainland counterparts, a project was put together by A.
								Shafer to investigate the degree of inbreeding in these populations. The project was done by Brooklyn Cars, an 
								undergraduate honours thesis student. I was involved through the co-supervision of Brooklyn's thesis, 
								but also through my native French as an interpreter and presenter during community outreach on SPM 
								(read more <a href="index.html#SPM">here</a>).
								The paper is in review, it is currently only  available as preprint on BioRxiv, here is the full abstract:
							
							<p>When a population is isolated and composed of few individuals, genetic drift is the paramount evolutionary 
								force. Inbreeding might also occur, resulting in genomic regions that are identical by descent, manifesting 
								as runs of homozygosity (ROHs) and the expression of recessive traits; accordingly, ROH can be used to map 
								phenotypic traits. Likewise, the genes underlying traits of interest can be revealed by comparing fixed SNPs and 
								divergent haplotypes between affected and unaffected individuals. Populations of white-tailed deer (Odocoileus 
								virginianus) on islands of Saint Pierre and Miquelon (SPM, France) have high incidences of leucism and 
								malocclusions; on the Florida Keys islands (USA), deer exhibit smaller body sizes. Here we aimed to reconstruct 
								island demography and identify the genes associated with these traits using ROHs, fixed SNPs, and divergent 
								haplotypes as markers in a pseudo case-control design. The two island populations of interest showed reduced 
								levels of genomic diversity, a low Ne, and a build-up of genetic load with significant divergence in Key deer. 
								Key deer showed higher inbreeding levels, but not longer ROHs, consistent with long-term isolation and a 
								historical bottleneck. We identified multiple trait-related genes in ROHs including LAMTOR2 which has links to 
								hypopigmentation and HES1 which is involved in the shortening of the maxilla and mandible. Our mixed approach of 
								linking ROH, fixed SNPs and haplotypes matched a high number (~50) of a-priori body size candidate genes in Key 
								deer. All three phenotypes showed patterns consistent with a complex trait and non-mendelian inheritance.</p>
							 
							<footer>
								<ul class="actions">
									<li><a href="http://dx.doi.org/10.1101/2023.08.01.551454" class="button large">Read the preprint</a></li>
								</ul>
							</footer>

						</article>						
					</div>

				<!-- Footer -->
					<section id="footer">
						<ul class="icons">
							<li><a href="https://github.com/CamilleKessler" class="icon brands fa-github"><span class="label">Github</span></a></li>
							<li><a href="https://gitlab.com/WiDGeT_TrentU/graduate_theses/-/tree/master/Kessler?ref_type=heads" class="icon brands fab fa-gitlab"><span class="label">Github</span></a></li>
							<li><a href="https://www.researchgate.net/profile/Camille-Kessler" class="icon brands fa-researchgate"><span class="label">Email</span></a></li>
							<li><a href="https://scholar.google.com/citations?user=uZYOLxAAAAAJ&hl=en" class="icon brands fab fa-google"><span class="label">Email</span></a></li>

						</ul>
						<p class="copyright">&copy; Camille Kessler. Design: <a href="http://html5up.net">HTML5 UP</a>.</p>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>